recipes/py_mini_racer/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "py_mini_racer" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/py_mini_racer-{{ version }}.tar.gz
+  sha256: f71e36b643d947ba698c57cd9bd2232c83ca997b0802fc2f7f79582377040c11
+
+build:
+  skip: True  # [linux]
```

Comment:
I'm a bit confused b/c the package looks like noarch but the sdist is shipping compiled code :scream: 

You probably need to figure out how to compiled the extension in https://github.com/sqreen/PyMiniRacer/tree/v0.6.0/py_mini_racer/extension.

I could not find any instructions in the docs. Also, it would be nice to run the full test suite in this case.

Reply:
Looks like the extension is built with https://github.com/sqreen/PyMiniRacer/blob/v0.6.0/helpers/v8_build.py

Note that there some download and patching that may not be ok for conda-forge. Ideally we should use the dependencies (V8?) already in conda-forge to ensure consistency with other packages. Not an easy feat!